OpenRA.Mods.Common/Activities/Air/Land.cs

```diff
@@ -44,7 +44,7 @@ public override Activity Tick(Actor self)
 				return NextActivity;
 			}
 
-			Fly.FlyToward(self, plane, d.Yaw.Facing, WDist.Zero);
+			Fly.FlyToward(self, plane, d.Yaw.Facing, plane.Info.LandAltitude);
```

Comment:
Good idea, I used to have target.actor.plane.cruiseHeight for the target height.